src/routes/forgotPassword.js

```diff
@@ -1,3 +1,56 @@
-export default(apiServer) => {
+import { list, patchOne } from 'mongoose-crudl'
+import AdminModel from '../models/Admin.js'
+import Email from '../models/Email.js'
+import allowAccessTo from 'bearer-jwt-auth'
+import { AuthenticationError, ValidationError } from 'standard-api-errors'
+import jwt from 'jsonwebtoken'
+import crypto from 'crypto'
 
+export default (apiServer) => {
+  const secrets = process.env.SECRETS.split(' ')
+
+  apiServer.post('/v1/forgot-password/send', async req => {
+    const response = await list(AdminModel, req.body, req.query)
+    if (response.result.count === 0) {
+      throw new AuthenticationError('Check user name')
+    }
+    const payload = {
+      type: 'forgot-password',
+      user: {
+        _id: response.result.items[0]._id,
+        email: response.result.items[0].email
+      }
+    }
+    const token = 'Bearer ' + jwt.sign(payload, secrets[0])
+    Email('example@example.com', 'forget password link ', `<h1>here is ur token: ${token}</h1>`)
```

Comment:
Please read in the forgot-password.html email template from the email-templates folder and use that one. If we manage our transactional email html templates there, our life will be easier.